### ISSUE TITLE
SideBarEnhancements not fount via Package Control

### ISSUE THREAD

**Issue by raedlime** [titoBouzout/SideBarEnhancements#327]
Hey,
hoping not to open multiple issues, I'm asking for help about installing SideBarEnhancements.

I reinstalled ST3 just a few minutes ago and couln't find the plugin via Package Control.
Is there a mistake on my part, or is there a real issue?

Hope you guys can help me out!

Thanks in advance.

**Comment by PortableStick** [comment]
I'm having the same issue.  Is the only way to install now manually through the repo?

**Comment by ypyang237** [comment]
Having the same issue, too. However, just follow the installation manually here : https://github.com/titoBouzout/SideBarEnhancements

Note: Make sure the package is named SideBarEnhancements not SideBarEnhancements-ST3

**Comment by alexblunck** [comment]
It looks like the package was "removed" from the Package Control registry along with all packages from titoBouzout.

![screen shot 2016-09-14 at 13 01 35](https://cloud.githubusercontent.com/assets/614291/18509888/7add704a-7a7b-11e6-8d83-9da97d5e4d6e.png)

**Comment by wiredearp** [comment]
I have the same problem. Viva titoBouzout!

**Comment by titoBouzout** [comment]
Yes, I had problems to add new packages to the repository because there's a "new"(has some time there) requirement to maintain a version number, and I don't want to maintain versions numbers, because to me a date is enough already.  I attempted to automate the version number by adding something without any meaning, just a date, but that broke the complete repository because "large number".  So because of this disagreement, I removed all my packages from there.  Rules changed, so Im no longer interested to distribute my packages via that channel. However you may decide to add this package manually via the option "Add Repository", this may or may not work, Im not uptodate with the requirements of that to work, and as I say, Im not interested.

**Comment by FichteFoll** [comment]
"Add Repository" will continue to work.

**Comment by colthreepv** [comment]
@FichteFoll actually, it doesn't if you don't specify the `st3` branch (this repo doesn't use `master`)

Just go in **Package Control: Add Repository** => `https://github.com/titoBouzout/SideBarEnhancements/tree/st3`

and package control will resume it's work.
[Credits: skuroda](http://stackoverflow.com/a/17901993/1071793)

**Comment by FichteFoll** [comment]
Ah, right. Tito's repos are a bit weird. Thanks for the heads up.

**Comment by braver** [comment]
@FichteFoll previously similar packages have been blocked from package control to prevent duplication. Just curious, would you now accept a package similar to this now that it is no longer listed?

**Comment by shobhitg** [comment]
@braver Definitely! If someone could clone this and make it available on Package Control, that would be awesome.

**Comment by FichteFoll** [comment]
@braver, yes of course. Package Control is only about the packages currently listed on it. If a package is removed, that functionality becoms available for a new package to take.

You could also fork the original package and continue developing it under the same name so that all the 100k users of the original SideBarEnhancements package will continue to receive updates.

Just note that we're currently very slow with reviewing package additions.

**Comment by braver** [comment]
@FichteFoll Thanks for the feedback. I don't necessarily intend to fork and publish this, although it's good to know that that would work and would even automatically migrate existing users. I might though, who knows what will happen.

**Comment by titoBouzout** [comment]
@FichteFoll I dont think is good idea to reuse this or any other package name again. People will believe they installing this, or you just taking over my own package. they may also will believe is this package and they installed the other one. I didnt give up to the package, I gave up to the distribution channel. This topic is over.
